client/src/favoriteButton.js

```diff
@@ -0,0 +1,28 @@
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+import FavoriteIcon from '@material-ui/icons/Favorite'; 
+
+class LikeButton extends React.Component {
```

Comment:
Better to write this as functional component instead of class component to be consistent.